hack/python-sdk/main.go

```diff
@@ -26,15 +26,22 @@ import (
 	"k8s.io/klog"
 	"k8s.io/kube-openapi/pkg/common"
 
-	mpijob "github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v1"
+	mpijobv1 "github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v1"
+	mpijobv2 "github.com/kubeflow/mpi-operator/v2/pkg/apis/kubeflow/v2beta1"
 )
 
 // Generate OpenAPI spec definitions for MPIJob Resource
 func main() {
-	if len(os.Args) <= 1 {
-		klog.Fatal("Supply a version")
+	if len(os.Args) <= 2 {
+		klog.Fatal("Supply the MPIJob version and the swagger spec version")
```

Comment:
The swagger spec version parameter exists already, this time only added the version api so that we can generate V1 or V2 sdk separately. So which parameter do you think can be removed?